taxi/forms.py

```diff
@@ -0,0 +1,54 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+
+
+class LicenseNumberValidationMixin(forms.ModelForm):
+    LENGTH = 8
+    FIRST_LETTERS = 3
+    LAST_DIGITS = 5
+
+    def clean_license_number(self) -> str:
+        license_number = self.cleaned_data["license_number"]
+        if len(license_number) != self.LENGTH:
+            raise forms.ValidationError(
+                f"License number length should be {self.LENGTH} characters"
+            )
+        if (
+            not license_number[: self.FIRST_LETTERS].isupper()
+            or not license_number[: self.FIRST_LETTERS].isalpha()
```

Comment:
not very sensitive, but if you work with same slice twice, consider giving it a name and make 2 checks, instead of making slice 2 times.